internal/proxy/connection/manager.go

```diff
@@ -42,7 +42,7 @@ func (s *connectionManager) Stop() {
 func (s *connectionManager) checkLoop() {
 	defer s.wg.Done()
 
-	t := time.NewTicker(paramtable.Get().ProxyCfg.ConnectionCheckIntervalSeconds.GetAsDuration(time.Second))
+	t := time.NewTimer(paramtable.Get().ProxyCfg.ConnectionCheckIntervalSeconds.GetAsDuration(time.Second))
```

Comment:
Need reset timer if use it instead of ticker

Reply:
You can expand the code below, the reset logic already exists before